src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -568,8 +564,6 @@ protected virtual async Task HandleWebSocketAsync(HttpContext context, RequestDe
         await selectedHandler.ExecuteAsync(context, socket, selectedProtocol, _userContextBuilderForWebSockets);
     }
 
-    *****************************/
```

Comment:
Regarding `  if (socket.SubProtocol != selectedProtocol)` conditions. Is such a situation possible in reality?

Reply:
I don’t know actually. It seems likely not but this validation check was in the old code and I didn’t see a good reason to remove it.